cmd/snap/cmd_snap_op.go

```diff
@@ -176,29 +176,61 @@ func (x *cmdInstall) Execute([]string) error {
 		name = snapName
 	}
 
-	return listSnaps([]string{name})
+	return listSnaps(&client.ListOptions{
+		Names: []string{name},
+	})
 }
 
 type cmdRefresh struct {
 	Channel    string `long:"channel" description:"Refresh to the latest on this channel, and track this channel henceforth"`
 	Positional struct {
 		Snap string `positional-arg-name:"<snap>"`
-	} `positional-args:"yes" required:"yes"`
+	} `positional-args:"yes"`
 }
 
-func (x *cmdRefresh) Execute([]string) error {
+func refreshAll() error {
+	// FIXME: move this to snapd instead and have a new refresh-all endpoint
 	cli := Client()
-	name := x.Positional.Snap
-	opts := &client.SnapOptions{Channel: x.Channel}
-	changeID, err := cli.Refresh(name, opts)
+	updates, err := cli.List(&client.ListOptions{Refresh: true})
```

Comment:
@pedronis Yeah, @chipaca pointed this out too, I was planning to do a followup branch for that, however if you think it is a blocker it can also be done now.